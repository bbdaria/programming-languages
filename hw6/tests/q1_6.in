use_module(library(yall)).
set_prolog_flag(answer_write_options, [quoted(true), portray(true), max_depth(0), spacing(next_argument)]).
print("===TEST START===").

{L}/(
    findall(
        [X],
        sort_me([5, -2, 10, 14, 1, 4, 0], X),
        L1
    ),
    msort(L1, L)
).