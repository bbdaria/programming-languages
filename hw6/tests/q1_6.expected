true.


L = [[[-2, 0, 1, 4, 5, 10, 14]]].


